src/components/molecules/NavBar/NavBar.js

```diff
@@ -1,25 +1,35 @@
-import React from "react";
-import Toolbar from '@material-ui/core/Toolbar';
-import ImageLink from '../../atoms/ImageLink/ImageLink';
-import { Grid } from '@material-ui/core';
-import NavigationLink from "../../atoms/NavigationLink/NavigationLink";
-import LanguageSelector from "../../atoms/LanguageSelector/LanguageSelector";
-
+import React from "react"
+import ImageLink from "../../atoms/ImageLink/ImageLink"
+import { Grid } from "@material-ui/core"
+import NavigationLink from "../../atoms/NavigationLink/NavigationLink"
+import LanguageSelector from "../../atoms/LanguageSelector/LanguageSelector"
 
 const NavBar = () => {
   const navigationData = [
-    {slug: "What we do", caption: "What we do"},
-    {slug: "Our Talent", caption: "Our Talent"},
-    {slug: "Contact Us", caption: "Contact Us"},
-  ];
+    { slug: "What we do", caption: "What we do" },
+    { slug: "Our Talent", caption: "Our Talent" },
+    { slug: "Contact Us", caption: "Contact Us" },
+  ]
 
   return (
-    <Toolbar>
-      <ImageLink slug="Image" imageUrl="https://images.ctfassets.net/bozygz3awzku/ylV3yo9Ruh411GawGTD2m/7ef1b7e910602d94fb74da0da6155178/ioet-logo9.png" />
-      <Grid container alignItems="flex-start" justify="flex-end" direction="row">
-        { navigationData.map(link =>
-          <NavigationLink slug={link.slug} caption={link.caption} style={{ align: "right"}} />
-        )}
+    <React.Fragment>
```

Comment:
@eacarras Please let me know if you have questions about this... I've considered moving the Toolbar tag to the header component.